tests/integration/cattletest/core/test_link.py

```diff
@@ -2,23 +2,24 @@
 
 
 @pytest.fixture(scope='module')
-def link_network(admin_client, sim_context):
-    nsp = create_agent_instance_nsp(admin_client, sim_context)
-    create_and_activate(admin_client, 'linkService',
+def link_network(internal_test_client, sim_context):
+    nsp = create_agent_instance_nsp(internal_test_client, sim_context)
+    create_and_activate(internal_test_client, 'linkService',
                         networkServiceProviderId=nsp.id,
                         networkId=nsp.networkId)
 
-    return admin_client.by_id_network(nsp.networkId)
+    return internal_test_client.by_id_network(nsp.networkId)
 
 
-def test_link_instance_stop_start(admin_client, sim_context, link_network):
-    target1 = create_sim_container(admin_client, sim_context,
+def test_link_instance_stop_start(internal_test_client, sim_context,
```

Comment:
test_link, test_ha, test_host_only, ip_pool, network_service_provider, virtual_machine are ok to use internal_test_client as discussed with @ibuildthecloud yesterday.